src/main/webapp/app/exercises/shared/exercise-headers/header-participation-page.component.ts

```diff
@@ -56,6 +56,12 @@ export class HeaderParticipationPageComponent implements OnInit, OnChanges {
             this.exerciseStatusBadge = hasExerciseDueDatePassed(this.exercise, this.participation) ? 'bg-danger' : 'bg-success';
             this.exerciseCategories = this.exercise.categories || [];
             this.dueDate = getExerciseDueDate(this.exercise, this.participation);
+            if (this.participation?.results?.[0]?.rated) {
```

Comment:
Q: Is the result at index 0 always the latest (rated) result?

Reply:
Practice runs are now in their own participation